cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -84,6 +91,29 @@ def _create_perfect_calibration(device: cirq.Device) -> calibration.Calibration:
     return calibration.Calibration(calibration=snapshot, metrics=all_metrics)
 
 
+def load_median_device_calibration(processor_id: str) -> calibration.Calibration:
+    """Loads a median Calibration for the given device.
+
+    Real calibration data from Google's 'rainbow' and 'weber' devices has been
+    saved in Cirq. The calibrations selected are roughly representative of the
+    median performance for that chip.
```

Comment:
Worth commenting (or linking to the docs) which metrics are included in this data?

Reply:
Maybe we should augment the docs then in a later PR.